internal/cmd/groups.go

```diff
@@ -176,3 +179,123 @@ $ infra groups remove Engineering`,
 
 	return cmd
 }
+
+func newGroupsAddUserCmd(cli *CLI) *cobra.Command {
```

Comment:
Should adding to a group have a force flag also?

Reply:
Hmm.. would this just create the user and add them to the group even though the user doesn't exist yet? I'm assuming the same would happen if the group which was specified didn't exist either.

The issue here would be that the API requires you to specify the user ID for the user (and not the user name). There would have to be some heavy(-ier) lifting in the client, or refactoring how the API endpoint works. I think that functionality would probably be best in a follow on PR.